# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ipt_sysrq/hotfix.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/rsysrq.h.orig	2007-07-30 13:54:47.000000000 +0000
+++ ./src/rsysrq.h	2007-07-30 13:56:57.000000000 +0000
@@ -2,10 +2,9 @@
 #ifndef _RSYSRQ_H
 #define _RSYSRQ_H
 
-#include <asm/byteorder.h>
-
 #ifndef __KERNEL__
 #include <sys/types.h>
+#include <asm/byteorder.h>
 
 typedef __uint8_t	u8;
 typedef __uint32_t	u32;
@@ -22,6 +21,7 @@
 #endif
 
 #else
+#include <asm/byteorder.h>
 #include <linux/string.h>
 #endif
 
